.github/actions/build-dotnet/action.yml

```diff
@@ -33,6 +33,14 @@ inputs:
       Change this to any other value to alter the cache key, effectively invalidating the latest cache. This is the only
       current way to force "clear" the cache (https://github.community/t/how-to-clear-cache-in-github-actions/129038/5)
       until the associated issue (https://github.com/actions/cache/issues/2) is resolved.
+  publish-version:
+    required: false
+    default: "1.${{ github.run_number }}.${{ github.run_attempt }}-${{ github.run_id }}"
```

Comment:
Does this really get evaluated as a default?

Reply:
Yup, like [here](https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/runs/7103788399?check_suite_focus=true#step:5:60).

![image](https://user-images.githubusercontent.com/4007293/176335237-bc1183b6-26db-4395-b431-1ad6a6855a8d.png)